### ISSUE TITLE
github_worker tries to write `False` into the `bigint` column `actor.site_admin`

### ISSUE THREAD

**Issue by maxhbr** [chaoss/augur#1333]
Please help us help you by filling out the following sections as thoroughly as you can.

**Description:**
While running, one can observe 
- error messages in the scheduler, that tells that the `github_worker` failed
- an corresponding error log in the DB: `invalid input syntax for type bigint: "False"`

**How to reproduce:**
1. use https://github.com/chaoss/augur/commit/aefae226106703fa5e1053802a0cefdfa9537d91
1. start docker compose augur setup (with self-build containers from source)
2. wait for the error to appear
3. Scroll down to:

```
[...]
backend_1   | [424] augur.routes.broker [ERROR] workers.github_worker.57840 ran into error while completing task: {'job_type': 'MAINTAIN', 'models': ['issues'], 'display_name': 'issues model for url: https://github.com/pytorch/cpuinfo.git', 'given': {'github_url': 'https://github.com/pytorch/cpuinfo.git'}, 'focused_task': 1, 'worker_id': 'workers.github_worker.57840'}
database_1  | 2021-07-09 07:06:53.589 UTC [303489] ERROR:  invalid input syntax for type bigint: "False"
database_1  | 2021-07-09 07:06:53.589 UTC [303489] CONTEXT:  COPY merge_data_0_960400, line 1, column actor.site_admin: "False"
database_1  | 2021-07-09 07:06:53.589 UTC [303489] STATEMENT:  COPY merge_data_0_960400 ("event", "issue.number", "issue.title", "issue.user", "actor.url", "issue.events_url", "issue.assignees", "actor.gravatar_id", "issue.node_id", "actor.organizations_url", "issue.updated_at", "issue.performed_via_github_app", "issue.created_at", "issue.html_url", "actor.starred_url", "actor.type", "issue.url", "issue.comments_url", "actor.followers_url", "actor.login", "issue.comments", "issue.assignee", "issue.pull_request", "actor.id", "issue.labels", "issue.state", "issue.body", "issue.milestone", "actor.node_id", "issue.author_association", "actor.avatar_url", "issue.labels_url", "actor.gists_url", "actor.html_url", "issue.repository_url", "actor.received_events_url", "issue.locked", "issue.active_lock_reason", "actor.subscriptions_url", "issue.id", "actor.events_url", "actor.site_admin", "actor.repos_url", "issue.closed_at", "actor.following_url", "actor") FROM STDIN WITH CSV
[...]
```

This can be found multiple times in the log.

**Software versions:**
 - Augur: https://github.com/chaoss/augur/commit/aefae226106703fa5e1053802a0cefdfa9537d91
 - OS: docker-compose

**Comment by sgoggins** [comment]
@IsaacMilarky : Can you make this fix?

**Comment by IsaacMilarky** [comment]
I remember seeing this error a while ago. However, I have not seen this error happen since we refactored how we enrich contributor data. I will continue trying to replicate it on the latest docker builds but I haven't seen this error in a while. There was also an issue with the docker container having to do with go preventing the container from building that has been fixed so the latest in dev should be able to be pushed to dockerhub soon. 

When I last looked into it, I remember it being caused by trying to insert enriched contributors so I feel like that might have been nipped by the refactor. 

I will keep trying to replicate the issue to be thorough.
